libraries/dawg-show/src/main/java/com/comcast/video/dawg/show/PowerController.java

```diff
@@ -89,7 +94,14 @@ public void power(@RequestParam(value="deviceIds[]") String[] deviceIds, @Reques
     }
 
     protected PowerClient getPowerClient(MetaStb stb, Cookie authCookie) {
-        PowerClient pc = new PowerClient(stb);
+        MetaStb meta = new MetaStb(new HashMap<String, Object>(stb.getData())); // make a copy so we can set some defaults
```

Comment:
it seems like this block is repeated 3 times... do you think it would be a good idea to factor it out? Maybe have a new class `DefaultMetaStb`?